redistimeseries/client.py

```diff
@@ -42,11 +42,19 @@ def parse_m_range(response):
                                 parse_range(item[2])]})
     return res
 
+def parse_get(response):
+    if response[0] == None:
```

Comment:
@ashtul the LGTM bot complains about this because python linting rules prefer to use `if response[0] is None`.